installer/scripts/omsadmin.sh

```diff
@@ -301,11 +320,7 @@ onboard()
         clean_exit 1
     fi
 
-
-    # If a test is not in progress then create workspace-specific directories
-    if [ -z "$TEST_WORKSPACE_ID" -a -z "$TEST_SHARED_KEY" ]; then
-        create_workspace_directories $WORKSPACE_ID 
-    fi
+    create_workspace_directories $WORKSPACE_ID
```

Comment:
Yes, the unittest and systemtest all passed with the new folder structure for multi-homing.